<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home</class>
 <widget class="QWidget" name="Home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="src.qrc">:/images/Home.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3,5">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Thông báo Thư viện</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="statusTip">
             <string>Chọn một  mục để xem chi tiết</string>
            </property>
            <property name="text">
             <string>Xem chi tiết</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="tableView_thongbao"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Tìm sách</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QComboBox" name="comboBox">
            <property name="statusTip">
             <string>Tìm kiếm theo</string>
            </property>
            <item>
             <property name="text">
              <string>ISBN</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Tất cả sách</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Tựa đề</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Tác giả</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>NXB</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Năm</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="src.qrc">
              <normaloff>:/images/search-icon.png</normaloff>:/images/search-icon.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>21</width>
              <height>21</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_muon">
            <property name="text">
             <string>Mượn sách</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="tableView"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
